test/e2e/positive.go

```diff
@@ -50,4 +71,98 @@ var _ = ginkgo.Describe("[PositiveFlow] Deployer validation", func() {
 			gomega.Expect(vo.Errors).To(gomega.BeEmpty())
 		})
 	})
+
+	ginkgo.Context("with cluster with all the deployed components", func() {
+
+		ginkgo.It("should have API components", func() {
+			mf, err := api.GetManifests()
+			gomega.Expect(err).ToNot(gomega.HaveOccurred())
+
+			key := getObjKey(mf.Crd.Name, mf.Crd.Namespace)
+			err = hp.WaitForObjectToBeCreated(key, &v1.CustomResourceDefinition{})
+			gomega.Expect(err).ToNot(gomega.HaveOccurred())
+		})
+
+		ginkgo.It("should have all RTE components", func() {
+			mf, err := rte.GetManifests()
+			gomega.Expect(err).ToNot(gomega.HaveOccurred())
+			mf = mf.UpdateNamespace()
+
+			for _, obj := range mf.ToObjects() {
+				key := getObjKey(obj.GetName(), obj.GetNamespace())
+				err = hp.WaitForObjectToBeCreated(key, obj)
+				gomega.Expect(err).ToNot(gomega.HaveOccurred(), fmt.Sprintf("object of kind %v was not created", obj.GetObjectKind()))
```

Comment:
I'm not sure we need this level of detail at this stage. But if we want to go at this level of detail (and again I don't think we need - now) we should also check the objects all have the right status (e.g. all ready, no errors)